helm-chart/banzai/requirements.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: postgresql
+  repository: https://kubernetes-charts.storage.googleapis.com
```

Comment:
Does this address Ira's recent comments that the locations of charts has moved?

Reply:
Fixed this.